src/main/resources/swagger/api-docs.yaml

```diff
@@ -223,7 +223,7 @@ paths:
   '/api/cluster/{googleProject}/{clusterName}':
     patch:
       summary: Updates the configuration of a cluster.
-      description: Cluster must be running. Learn more about scaling clusters here: https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/scaling-clusters
+      description: In order to update the configuration of a cluster, it must first be running
```

Comment:
i decided the link was confusing because it made it seem like users could/should update their clusters outside of leo